src/jit/compiler.hpp

```diff
@@ -632,7 +632,7 @@ bool                isRegParamType(var_types type)
 #endif // !_TARGET_X86_
 }
 
-#ifdef _TARGET_AMD64_
+#if defined(_TARGET_AMD64_) || defined(_TARGET_ARM64_)
```

Comment:
My understanding is `argOrReturnTypeForStruct` handles arm64 for multi-reg case.